### ISSUE TITLE
Normalize family names

### ISSUE THREAD

**Issue by cthoyt** [indralab/gilda#22]
I'm not sure how many special cases you want to add in to Gilda, but there are lots of times when text comes with the suffix "family" like "RAS family" that would be easy to normalize to an appropriate term like `fplx:RAS`

**Comment by bgyori** [comment]
Yes, this could be handled in two ways: either when generating lookup strings, for an input like "RAS family", we could also look up "RAS". Alternatively, * family and * complex could be added to all FamPlex entries as synonyms.

**Comment by cthoyt** [comment]
Hmm i think adding family and complex into the synonyms would probably be the simplest and also most informative
